CMakeLists.txt

```diff
@@ -138,10 +138,19 @@ option(GOOGLE_CLOUD_CPP_ENABLE_FIRESTORE
        "Enable building the Firestore library." ON)
 option(GOOGLE_CLOUD_CPP_ENABLE_PUBSUB "Enable building the Pub/Sub library." ON)
 
+# Compute the default value for GOOGLE_CLOUD_CPP_ENABLE_GRPC_UTILS, use a
+# function to create a scope for the temporary variables.
```

Comment:
nit: this comment about using a function no longer applies.

Reply:
Argh, will send a new PR to fix that.